01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -57,4 +57,4 @@ public function getName()
     public function celebrateBirthday() {
         echo 'Yay!';
     }
-} 
\ No newline at end of file
+}
```

Comment:
You seem to have removed newlines at the end of several files. There should be a newline (see, for example; http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)